identity/src/main/kotlin/net/thechance/identity/security/JwtService.kt

```diff
@@ -38,4 +50,21 @@ class JwtService(
 
         return UUID.fromString(subject)
     }
+
+    fun isAdminToken(token: String): Boolean {
+        val claims = Jwts.parserBuilder()
+            .setSigningKey(secretKey)
+            .build()
+            .parseClaimsJws(token)
+            .body
+
+        return when (val adminClaim = claims[ADMIN_CLAIM]) {
+            is Boolean -> adminClaim
+            else -> false
+        }
```

Comment:
why not make it like this ?

```suggestion
        return claims[ADMIN_CLAIM] as? Boolean ?: false
```

Reply:
Ok I was trying to avoid the type casting but no problem

Reply:
it's normal in this case and the code will be easier to understand if we use the type casting, why do you hate the casting 😂

Reply:
I have previous problems with the casting 😂 
it can throw unexpectedly in some cases